### PR TITLE
Update format of custom_extras files

### PR DESCRIPTION
**Pull request by bryteise**
Switching to flat file handling using the same conf file parsing as
other config files like extras and dev_extras.

Moves to two different config files for the file list and the requires
list.

### CODE REVIEW

File: autospec/git.py [hunk 1]
```diff
@@ -60,7 +60,6 @@ def commit_to_git(path):
     call("bash -c 'shopt -s failglob; git add -f *.sha256'", check=False, stderr=subprocess.DEVNULL, cwd=path)
     call("bash -c 'shopt -s failglob; git add -f *.sign'", check=False, stderr=subprocess.DEVNULL, cwd=path)
     call("bash -c 'shopt -s failglob; git add -f *.pkey'", check=False, stderr=subprocess.DEVNULL, cwd=path)
-    call("bash -c 'shopt -s failglob; git add -f *_extras'", check=False, stderr=subprocess.DEVNULL, cwd=path)
```
Comment (phmccarty) [inline autospec/git.py]: Don't we still want to commit any files that match `*_extras`? Though it needs an extension to `*_extras*` now.

Comment (bryteise) [reply autospec/git.py]: This is something that is handled via read_conf_file, when foo_extras and foo_extras_requires is read they are added to the set of config_files that git manages.

Comment (phmccarty) [reply autospec/git.py]: Ah, yes, I see that now, thanks.

**Comment by bryteise** [comment]
@thiagomacieira This should make life easier for updating the extras files.

**Pull request merged**